Client/Ecosia/UI/APNConsent/APNConsentViewController.swift

```diff
@@ -204,13 +205,18 @@ extension APNConsentViewController {
 extension APNConsentViewController {
 
     @objc private func skipTapped() {
+        Analytics.shared.apnConsent(.skip)
         dismiss(animated: true)
     }
 
     @objc private func ctaTapped() {
         guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else { return }
         ClientEngagementService.shared.requestAPNConsent(notificationCenterDelegate: appDelegate) { granted, error in
-            // TODO: Add analytics
+            guard granted else {
+                Analytics.shared.apnConsent(.deny)
```

Comment:
Btw, I think those are outdated in [the ticket](https://ecosia.atlassian.net/browse/MOB-2060), I see there it's still `click` instead of `allow/deny`

Reply:
The ticket is, in fact, outdated 🤦 . Confluence has the most up-to-date Analytics where `allow/deny` are the intended ones. Will double-check the Ticket and grab the updated values from confluence! THanks!